articles/api-auth/using-rules/_includes/_create-rule.md

```diff
@@ -0,0 +1,15 @@
+Create a file named `myrule.js`, and enter the following:
+
+```js
+module.exports = function(${args}) {
+  var access_token = {};
+  access_token['https://foo.com/claim'] = 'bar';
+  access_token.scope = scope;
+  access_token.scope.push('extra');
+  cb(null, access_token);
+};
+```
+This is a sample rule that will:
+
+* Add an arbitrary claim (`https://foo.com/claim`) to the `access_token`.
```

Comment:
Cool, I'll add some info now on the doc and once the one you wrote is published I will link to that.